### ISSUE TITLE
Protolint linting Proto files in node_modules

### ISSUE THREAD

**Issue by ajimae** [yoheimuta/protolint#155]
Environment 
 - MacOS catalina

- Protolint Version 0.28.2

When I run protolint lint . I get a very weird error

```bash
found "\"package\"(Token=30, Pos=node_modules/grpc/node_modules/protobufjs/examples/protoify/json.proto:2:1)" but expected [syntax] at /go/pkg/mod/github.com/yoheimuta/go-protoparser/v4@v4.3.0/parser/syntax.go:59. Use -v for more details
```
Please I have been having this issues for some days now, I have tried ignoring the node_modules folder entirely in my protolint.yml config file but to no success.

Please help!

PS

This is the content of my `protolint.yml` file

```yaml
lint:
  files:
    exclude:
      - node_modules/grpc/node_modules/protobufjs/examples/protoify/json.proto
  directories:
    # The specific directories to exclude.
    exclude:
      # NOTE: UNIX paths will be properly accepted by both UNIX and Windows.
      # - identity-service-proto/node_modules/
      - node_modules/
  rules:
    no_default: false # we can choose if we want the default linting in addition to our custom lintings too
    all_default: false

    add:
      # - MESSAGE_NAMES_UPPER_CAMEL_CASE
      # - SERVICE_NAMES_UPPER_CAMEL_CASE
      # - MESSAGES_HAVE_COMMENT
      # - REPEATED_FIELD_NAMES_PLURALIZED
      # - SERVICES_HAVE_COMMENT
      # - ENUM_FIELD_NAMES_UPPER_SNAKE_CASE
      # - ENUM_NAMES_UPPER_CAMEL_CASE
      # - FIELD_NAMES_LOWER_SNAKE_CASE
      # - PACKAGE_NAME_LOWER_CASE
      # - RPC_NAMES_UPPER_CAMEL_CASE

    remove:
      - MAX_LINE_LENGTH
      - MESSAGES_HAVE_COMMENT
```

**Comment by yoheimuta** [comment]
@ajimae Thank you for reaching out.

I'm suspecting that the protolint couldn't load a yaml file properly.
Can you give it a try the below points one by one?

- Renaming to [.protolint.yaml](https://github.com/yoheimuta/protolint#configuring)?
  - ref. https://yaml.org/faq.html
- Specifying to your config file with `-config_path` option

```
~/w/p/g/s/g/y/p/tmp ❯❯❯ ls -la
total 27944
drwxr-xr-x   5 yoheimuta  staff       160 Feb 19 09:20 .
drwxr-xr-x  25 yoheimuta  staff       800 Feb 19 09:09 ..
-rw-r--r--   1 yoheimuta  staff       921 Feb 19 09:08 .protolint.yaml
drwxr-xr-x   3 yoheimuta  staff        96 Feb 19 09:07 node_modules
-rwxr-xr-x   1 yoheimuta  staff  14300668 Feb 19 09:08 protolint
~/w/p/g/s/g/y/p/tmp ❯❯❯ tree
.
├── node_modules
│   └── grpc
│       └── node_modules
│           └── protobufjs
│               └── examples
│                   └── protoify
│                       └── json.proto
└── protolint

6 directories, 2 files
~/w/p/g/s/g/y/p/tmp ❯❯❯ ./protolint .
~/w/p/g/s/g/y/p/tmp ❯❯❯ 
```

PS. I'm thinking of adding protolint.yml and .protolint.yml to the search candidates because the extension of yml is also widespread. Plus, it would be better to improve the protolint to print whether it loads the config file properly when the -v option is specified.

**Comment by yoheimuta** [comment]
> adding protolint.yml and .protolint.yml to the search candidates

I supported this feature in v0.29.0. #156

>  it would be better to improve the protolint to print whether it loads the config file properly when the -v option is specified.

I also included this support in v0.30.0. #157

**Comment by yoheimuta** [comment]
Then, I closed this issue for now.
